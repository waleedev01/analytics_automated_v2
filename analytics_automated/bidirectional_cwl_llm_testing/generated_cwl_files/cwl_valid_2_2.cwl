cwlVersion: v1.2
class: Workflow

inputs: {}
outputs: {}

steps:
  step1:
    run: commandlinetool.cwl
    in:
      input1: input_step1
    out:
      output1: output_step1

  step2:
    run: commandlinetool2.cwl
    in:
      input2: step1/output1
    out:
      output2: output_step2

baseCommand: []

File: commandlinetool.cwl
class: CommandLineTool
inputs: {}
outputs: {}
baseCommand: []

File: commandlinetool2.cwl
class: CommandLineTool
inputs: {}
outputs: {}
baseCommand: []