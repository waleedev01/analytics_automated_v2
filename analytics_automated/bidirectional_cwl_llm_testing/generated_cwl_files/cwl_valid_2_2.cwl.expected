{"is_valid": true, "error": "CWL file is valid.", "file_type": "valid", "file_name": "cwl_valid_2_2.cwl"}